sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -320,3 +355,50 @@ def __setattr__(self, key, value):
                 self.early_termination.slack_amount = None
             if self.early_termination.slack_factor == 0.0:
                 self.early_termination.slack_factor = None
+
+    @property
+    def early_termination(self) -> Union[str, EarlyTerminationPolicy]:
+        return self._early_termination
+
+    @early_termination.setter
+    def early_termination(self, policy_dict: Union[EarlyTerminationPolicy, Dict[str, Union[str, float, int, bool]]]):
+        if policy_dict is None:
+            self._early_termination = None
+        elif isinstance(policy_dict, EarlyTerminationPolicy):
+            self._early_termination = policy_dict
+        elif isinstance(policy_dict, dict):
+            assert 'type' in policy_dict.keys()
+            policy_type = policy_dict['type'].lower().capitalize()
+            if policy_type == EarlyTerminationPolicyType.BANDIT:
+                self._early_termination = BanditPolicy(
+                    delay_evaluation=policy_dict.get('delay_evaluation', 0),
```

Comment:
why can't this be implemented as schema reserialize?

Reply:
modified